src/Swashbuckle.AspNetCore.SwaggerGen/XmlComments/XmlCommentsExampleHelper.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Text.Json;
+using Microsoft.OpenApi.Any;
+using Microsoft.OpenApi.Models;
+
+namespace Swashbuckle.AspNetCore.SwaggerGen
+{
+    public static class XmlCommentsExampleHelper
```

Comment:
Make this internal, we don't need to expose new public APIs for this.

Reply:
done